src/main/java/com/lpvs/util/LPVSWebhookUtil.java

```diff
@@ -38,6 +38,9 @@ public static LPVSQueue getGitHubWebhookConfig(String payload) {
         webhookConfig.setHeadCommitSHA(json.getAsJsonObject("pull_request")
                                             .getAsJsonObject("head")
                                             .get("sha").getAsString());
+
+        webhookConfig.setSender(json.getAsJsonObject("sender").get("login").getAsString());
+        webhookConfig.setOwner(json.getAsJsonObject("pull_request").getAsJsonObject("head").getAsJsonObject("repo").getAsJsonObject("owner").get("login").getAsString());
```

Comment:
When I reopened this PR, `sender` is `tiokim` and `owner` is `kyudori`. Is it right?
I think `Samsung` and `kyudori` are important.
What about changing `sender` and `owner` to `pull_request_base` and `pull_request_head` like the code example below?

```suggestion

        webhookConfig.setPRBase(json.getAsJsonObject("pull_request").getAsJsonObject("base").getAsJsonObject("repo").getAsJsonObject("owner").get("login").getAsString());
        webhookConfig.setPRHead(json.getAsJsonObject("pull_request").getAsJsonObject("head").getAsJsonObject("repo").getAsJsonObject("owner").get("login").getAsString());
```

Reply:
Thinking about it again, `sender` is also important. 
What about changing `base` instead of `head`?
```suggestion

        webhookConfig.setSender(json.getAsJsonObject("sender").get("login").getAsString());
        webhookConfig.setOwner(json.getAsJsonObject("pull_request").getAsJsonObject("base").getAsJsonObject("repo").getAsJsonObject("owner").get("login").getAsString());
```

Reply:
As you reviewed, modifying the code to `base`, `head`, and `sender`.

Reply:
Commit as soon as the modification is completed.

Reply:
Done. 
Added `pull_request_base`, `pull_request_head` to the pull_requests table.